bindings/python/docs/source/installation.rst

```diff
@@ -0,0 +1,67 @@
+Installing / Upgrading
+======================
+.. highlight:: bash
+
+System Compatibility
+--------------------
+
+PyMongoArrow is regularly built and tested on macOS and Linux
+(Ubuntu 20.04).
+
+Python Compatibility
+--------------------
+
+PyMongoArrow is currently compatible with CPython 3.6, 3.7 and 3.8.
```

Comment:
And 3.9.

Reply:
Indeed... I hadn't put this in here since it wasn't mentioned in PyArrow docs but looks like they do support it.